### ISSUE TITLE
Adding an item to the world has a default velocity

### ISSUE THREAD

**Issue by DarkArc** [SpongePowered/SpongeCommon#147]
````java
Optional<Entity> optEntity = world.createEntity(EntityTypes.DROPPED_ITEM, pos);
if (optEntity.isPresent()) {
    Item item = (Item) optEntity.get();
    item.offer(item.getItemData().set(Keys.REPRESENTED_ITEM, stack));
    world.spawnEntity(item);
}
````
In a previous version of Sponge, the above code would result in an item being spawned at "pos" without being "thrown"/assigned a default velocity. Now, this results in the equivalent of a natural item spawn, where the item is thrown as if a block has been mined. 

While both functionalities (no velocity, and vanilla random spawn velocity) are desirable, this behavior seems strange for the above case.

**Comment by Zidane** [comment]
@DarkArc 

Is this still valid?

**Comment by DarkArc** [comment]
As far as I've seen yes.

**Comment by simon816** [comment]
This would have been an effect of #37
The velocity can easily be set to zero after creating the entity,
```java
item.offer(Keys.VELOCITY, Vector3d.ZERO);
```
The question is, when creating an entity should sensible defaults take priority over setting everything to zero?

**Comment by Zidane** [comment]
@simon816 

Not sensible defaults, whatever is happening in the entities' constructor by default in the base game is exactly the same we should be doing.

``` java
    public EntityItem(World worldIn, double x, double y, double z)
    {
        super(worldIn);
        this.health = 5;
        this.hoverStart = (float)(Math.random() * Math.PI * 2.0D);
        this.setSize(0.25F, 0.25F);
        this.setPosition(x, y, z);
        this.rotationYaw = (float)(Math.random() * 360.0D);
        this.motionX = (double)((float)(Math.random() * 0.20000000298023224D - 0.10000000149011612D));
        this.motionY = 0.20000000298023224D;
        this.motionZ = (double)((float)(Math.random() * 0.20000000298023224D - 0.10000000149011612D));
    }
```

Which means that you should set the velocity to ZERO as simon suggests @DarkArc after creating the Entity but before spawning it.

**Comment by DarkArc** [comment]
@Zidane I would like to bring a counter point that we should always go for the bare minimum behavior (nothing more than has been requested). 

When we introduce the constraint that we're going to be relying on the default construction of the object, that opens us up for our behavior to change unexpectedly between versions of Minecraft. 

A good example of how this can be a problem is one I encountered while using Bukkit. They "fixed" their mob spawning code by modifying the spawning methods to behave as a "normal" mob spawn would. However, this lead to problems with some of my plugins, as entities I intended -- and needed -- to be spawned "simple" were being spawned with potion effects/weapons/armor. 

I would instead suggest extending the API in such a way that we must opt-in to "vanilla X" behavior, by doing this, we can -- at least in theory, we're a moding API so there's always that -- provide stronger guarantees about the future behavior of the API -- those which we explicitly state, not what we inherit.
